woocommerce/woocommerce
# add support for grouped product to api package

This PR will remain draft until #29198 is merged. This PR separates out the price properties. That change will need to be applied to the external product support.

### Changes proposed in this Pull Request:

This PR adds support for the grouped product type to the `api` package. It includes a E2E test to verify the properties specific to the grouped product type.

Closes #28884 .

### How to test the changes in this Pull Request:

1. Check CI test runs.
2. Run tests locally and note the `REST API > Grouped Product` section
3. Edit the `grouped` product in `tests/e2e/config/default.json` and add more simple products to the `groupedProducts` array.
4. Manually run the api test `npx wc-e2e test:e2e tests/e2e/specs/rest-api/grouped-product.test.js`.
5. Repeat 3 & 4 with a few combinations. Check the dashboard after each test to verify the products were created correctly.

### Other information:


### Changelog entry

N/A
